src/components/PostEditor/NormalPostContent.tsx

```diff
@@ -114,19 +115,31 @@ export default function NormalPostContent({
 
   return (
     <div className="space-y-4">
+      {parentEvent && (
+        <div className="bg-muted rounded-xl p-3 border border-border">
+          <div 
+            className="max-h-48 overflow-y-auto" 
+            style={{ 
+              scrollbarWidth: 'thin',
+              scrollbarColor: 'rgba(156, 163, 175, 0.5) transparent'
+            }}
+          >
+            <Note
+              size="small"
+              event={parentEvent}
+              hideStats
+              hideParentNotePreview
+            />
+          </div>
+        </div>
```

Comment:
```
        <ScrollArea className="max-h-48 rounded-lg overflow-y-auto">
          <div className="p-2 sm:p-3 bg-muted pointer-events-none">
            <Note size="small" event={parentEvent} hideStats hideParentNotePreview />
          </div>
        </ScrollArea>
```

Use the `ScrollArea` to unify the styles. The `padding` should be inside the `ScrollArea`. Otherwise, there will be extra space at the top and bottom when scrolling. `pointer-events-none` is used to prevent the dialog from being closed when clicking on images or other contents in the embedded notes. I'm not sure why this is the case. I would be extremely grateful if you could solve this problem.

Reply:
```
        <ScrollArea className="max-h-48 rounded-lg border bg-muted/40 overflow-y-auto">
          <div className="p-2 sm:p-3 pointer-events-none">
            <Note size="small" event={parentEvent} hideStats hideParentNotePreview />
          </div>
        </ScrollArea>
```

Please use this code directly, as both the padding size and the corner radius need to be consistent with the existing design.

I have modified the background color. This is because when the replied note contains embedded notes, the border of the embedded notes needs to be displayed. 

![WX20250406-180007@2x](https://github.com/user-attachments/assets/e7ca50c3-e391-435a-96cf-df595ef3bb23)